spring-cloud-kubernetes-config/src/test/java/org/springframework/cloud/kubernetes/config/ConfigMapsSpringBootTest.java

```diff
@@ -75,11 +76,13 @@ public void beforeTest() {
 			.once();
 	}
 
+	@Ignore
```

Comment:
Do you want that we move this test 

```
	@Test
	public void testConfig() {
		assertEquals(config.getMasterUrl(),mockClient.getConfiguration().getMasterUrl());
		assertEquals(config.getNamespace(),mockClient.getNamespace());
	}
```
within another a new Test Class of the spring-cloud-kubernetes-core project ?